linux_os/guide/system/crypto_policies/configure_bind_crypto_policy/bash/shared.sh

```diff
@@ -0,0 +1,14 @@
+# platform =  multi_platform_fedora
+
+function remediate_bind_crypto_policy() {
+	CONFIG_FILE="/etc/named.conf"
+	if test -f "$CONFIG_FILE"; then
+		sed -i 's|options {|&\n\tinclude "/etc/crypto-policies/back-ends/bind.config";|' "$CONFIG_FILE"
```

Comment:
This remediation will put the include right after the `options {` which is not the same as description says:
```
...make sure that the following line is not commented out or superseded by later includes:
include "/etc/crypto-policies/back-ends/bind.config";
```
If you put the include at the beginning it can be superseded by later includes.